panel/models/tabulator.ts

```diff
@@ -145,7 +159,15 @@ function group_data(records: any[], columns: any[], indexes: string[], aggregato
   }
   const aggs = []
   for (const index of indexes) {
-    aggs.push((index in aggregators) ? aggregators[index] : "sum")
+    if (index in aggregators) {
+      if (aggregators[index] instanceof Map) {
+        aggs.push(Object.fromEntries(aggregators[index]))
```

Comment:
I haven't managed to get this line to execute, could you provide an example that executes it?

Reply:
```python
from bokeh.sampledata.population import data as population_data

pop_df = population_data[population_data.Year.isin([2000, 2020])]
pop_df = pop_df[pop_df.AgeGrp.isin(['0-4', '10-14'])]
pop_df = pop_df[pop_df.Location.isin(['Afghanistan', 'Angola'])]
pop_df = pop_df.pivot_table(index=["Location", "AgeGrp", "Sex"], columns="Year", values="Value")[[2000, 2020]].dropna()
nested_aggs = {'AgeGrp': {2000: "mean", 2020: "min"}}

pn.widgets.DataFrame(value=pop_df, hierarchical=True, aggregators=nested_aggs).servable()
```